bin/v-update-firewall

```diff
@@ -189,11 +189,11 @@ else
     preup="/etc/network/if-pre-up.d/hestia-iptables"
     # Recreate the Hestia iptables rules loading script
     rm -f $routable $preup
-    if dpkg-query -W -f'${Status}' "netplan*" 2>/dev/null | grep -q "ok installed" && [ -d /etc/netplan ] && [ -n "$(ls -A /etc/netplan 2>/dev/null)" ]; then
+    if dpkg-query -W -f'${Status}' "netplan.io" 2>/dev/null | grep -q "ok installed" && [ -n "$(ls -A /etc/netplan 2>/dev/null)" ] && systemctl is-enabled networkd-dispatcher >/dev/null 2>&1; then
         echo '#!/bin/sh' > $routable
         echo '' >> $routable
         echo 'if [ "$IFACE" = "'$(ip route list | awk '/default .+/ {print $5}' | uniq)'" ]; then' >> $routable
-        echo '    [ -x "'$(which ipset)'" ] && '"${HESTIA}/bin/v-update-firewall-ipset" >> $routable
```

Comment:
In https://github.com/hestiacp/hestiacp/blob/03898485c1266aefb1f94de490b65fa55795e223/bin/v-add-firewall-ipset#L39 and https://github.com/hestiacp/hestiacp/blob/main/bin/v-delete-firewall-ipset#L37 it will be the same. 

We should probably set it as an dependency in hestia package.

Reply:
Yes, so it's safe to remove check IPSet here.

Currently, "v-add-firewall-ipset" and "v-delete-firewall-ipset" handle the accidental loss of IPSet binary:
https://github.com/hestiacp/hestiacp/blob/main/bin/v-add-firewall-ipset#L69
https://github.com/hestiacp/hestiacp/blob/main/bin/v-delete-firewall-ipset#L45

I added check IPSet in https://github.com/hestiacp/hestiacp/pull/1858 to avoid long time blockage caused by the loss of IPSet binary when starting system.

Considering that the probability of this situation should be very low, and "which" may cause new issue, so I decided to remove it.

Also, "which" is an external call, we should consider review all Hestia shells and replace it with the built-in "command -v" of BASH.

Reply:
Oke